### ISSUE TITLE
Unable to connect via SSH

### ISSUE THREAD

**Issue by sangorrin** [future-architect/vuls#568]
# Environment

## Vuls

Hash : vuls v0.4.2 0e87360

## OS
- Target Server: Debian 8.9
- Vuls Server: Debian 8.9

## Go

Go version: 1.8.3

# Current Output
```
jenkins@fuego:~/go/src/github.com/future-architect/vuls$ cat config.toml 
[servers]
[servers.debian]
host        = "192.168.1.69"
port        = "22"
user        = "user"
keyPath     = "/fuego-ro/boards/id_rsa"
jenkins@fuego:~/go/src/github.com/future-architect/vuls$ vuls configtest -debug debian
[Dec 22 09:00:18]  INFO [localhost] Validating config...
[Dec 22 09:00:18]  INFO [localhost] Detecting Server/Container OS... 
[Dec 22 09:00:18]  INFO [localhost] Detecting OS of servers... 
[Dec 22 09:00:18] DEBUG [localhost] Executing... ls /etc/debian_version
[Dec 22 09:00:18] DEBUG [localhost] execResult: servername: debian
  cmd: /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none user@192.168.1.69 -p 22 -i /fuego-ro/boards/id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version
  exitstatus: 255
  stdout: 
  stderr: 
  err: %!s(<nil>)
[Dec 22 09:00:18] ERROR [localhost] (1/1) Failed: debian, err: [Failed to detect OS: Unable to connect via SSH. Check SSH settings. If you have never SSH to the host to be scanned, SSH to the host before scanning in order to add the HostKey. user@192.168.1.69 port: 22
execResult: servername: debian
  exitstatus: 255
  stdout: 
  stderr: 
  err: %!s(<nil>)]
[Dec 22 09:00:18] ERROR [localhost] Failed to init servers: No scannable servers
```
# Addition Details

If I execute the command manually it works:
```
jenkins@fuego:~/go/src/github.com/future-architect/vuls$ /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none user@192.168.1.69 -p 22 -i /fuego-ro/boards/id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version
/etc/debian_version
jenkins@fuego:~/go/src/github.com/future-architect/vuls$ /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none user@192.168.1.69 -p 22 -i /fuego-ro/boards/id_rsa -o PasswordAuthentication=no stty cols 1000; cat /etc/debian_version
8.9
```
## Expected Behavior

ssh should connect

## Actual Behavior

ssh connection fails

## Steps to reproduce the behaviour

Install last version of vuls, prepare config.toml as above and run "vuls configtest debian"

**Comment by kotakanbe** [comment]
If you run it again and get the same error, you may find something if you add -vvv.
 https://github.com/future-architect/vuls/blob/master/scan/executil.go#L273

**Comment by sangorrin** [comment]
Thanks for the tip. It seems that the reason was that I was running vuls from a daemon without a home directory and the file .known_hosts was not being generated. I solved it by running "ssh-keyscan -t ecdsa 192.168.1.69" >> /etc/ssh/ssh_known_hosts".
